src/main/java/io/jenkins/plugins/aws/global_configuration/AwsGlobalConfigurationCategory.java

```diff
@@ -26,12 +26,14 @@
 
 import hudson.Extension;
 import jenkins.model.GlobalConfigurationCategory;
+import org.jenkinsci.Symbol;
 
 /**
  * @author Carlos Sanchez
  * @since
  *
  */
+@Symbol("aws")
```

Comment:
Inferred by chomping off pieces of the `Class.simpleName`, but best to be explicit.